public/style.css

```diff
@@ -1,51 +1,132 @@
-    bottom: 0;
-    left: 0;
-    z-index: 100;
-    padding: 90px 0 0;
-    box-shadow: inset -1px 0 0 rgba(0, 0, 0, .1);
-    z-index: 99;
+
+
+:root, [data-bs-theme=light] {
+  --bs-breadcrumb-margin-bottom: 0.2rem;
 }
 
-@media (max-width: 767.98px) {
-    .sidebar {
-        top: 11.5rem;
-        padding: 0;
+/* Defaults. */
+/* for extra small screens, show a similar sidebar but full width and only if the menu button is pressed. */
+@media(max-width: 768px) {
+    button.navbar-toggler {
+        display: inline;
+        border-radius: 0;
+        border: 0;
+        background-color: var(--bs-secondary-bg);
+    }
+    nav#sidebar {
+        display: none;
+    }
+    nav#sidebar a.nav-link-extra { margin-top: 1em };
+    nav#sidebar a.nav-link {
+        display: block;
+        width: 100%;
+        font-size: 150%;
+        height: 40px;
+    }
+    nav#sidebar a.nav-link:click {
+        background-color: var(--bs-primary-bg);
+    }
+
+    nav#sidebar.show {
+        position: fixed;
+        display: flex;
+        flex-wrap: wrap;
+        align-content: start;
+        height: 100vH;
+        width: 220px;
+        top: 3rem;
+        right: 0;
+        padding: 1em;
+        background-color: var(--bs-secondary-bg);
+        z-index: 99;
+    }
+
+    div.mainarea {
+        position: relative;
+        top: 3rem;
     }
+
+    .card-body {
+        padding: calc(var(--bs-card-spacer-y) * 0.3);
+    }
+
 }
-    
-.navbar {
-    box-shadow: inset 0 -1px 0 rgba(0, 0, 0, .1);
+
+/* [>=sm] For screens between, show a top icon bar with or without labels. */
+@media(min-width:768px) {
+    button.navbar-toggler { display: none; }
+    nav#sidebar { display: flex; }
+    nav#sidebar a.nav-link {
+        padding-left: 0.5rem;
+        padding-right: 0.5rem;
+        padding-top: 0.2rem;
+    }
+    nav#sidebar a.nav-link-extra { display: none };
+    nav#sidebar a.nav-link span { display: none; }
+    div.mainarea { margin-top: 3rem; }
 }
 
-@media (min-width: 767.98px) {
-    .navbar {
-        top: -100;
-        position: sticky;
-        z-index: 999;
+/* [lg] */
+@media(min-width:992px) {
+    nav#sidebar a.nav-link span { display: inline; }
+
+}
+
+/* For extra wide screens, show a static sidebar at the left, with text, vertically aligned. */
+@media (min-width: 1200px) {
+    nav#sidebar {
+        position: fixed;
+        flex-wrap: wrap;
+        align-content: start;
+        height: 100vH;
+        width: 180px;
+        left: 0;
+        top: 3rem;
+        padding: 1em;
+        background-color: var(--bs-secondary-bg);
+        z-index: 99;
+    }
+    nav#sidebar a.nav-link {
+        padding-left: 0;
+        padding-right: 0;
+        padding-top: 0;
+    }
+    div.mainarea {
+        margin-left: 180px;
+        width: calc(100vW - 180px);
     }
-/*
-    .sidebar {
-        width: 160px;
-    }*/
 }
 
-.sidebar .nav-link {
-    color: #333;
+.breadcrumb-row {
+    background-color: var(--bs-secondary-bg);
```

Comment:
the background of the breadcrums are part of the main content and should thus have the same background color

Reply:
I find it hard to distinguish where the real content begins this way. 

Also, with a no-background card and with no visual clues what is what, there#s a lot of duplicated headings:
![image](https://github.com/antedebaas/DMARC-SMTPTLS-Reports/assets/44540/08b95cc2-8b43-4c64-b769-dfcadda724e4)

Reply:
the breadcrubs are a main navigator and are always placed on top of the content. it allows for easy navigating back

Reply:
OK, it's up to you. Feel free to revert this if this is the only change that blocks this PR.